openssl_ocsp_responder/ocsp_responder.py

```diff
@@ -69,6 +76,11 @@ def start_responder(self):
         self.responder_process = subprocess.Popen(args=args,
                                                   stdout=subprocess.PIPE,
                                                   stderr=subprocess.STDOUT)
+
+        if self.get_status(self.ocsp_certificate_path, self.ca_certificate_path, self.request_timeout) is None:
+            self.stop_responder()
+            self.delete_crl_file()
+            raise Exception("Failed to get a response from the responder")
```

Comment:
Please define your own exception type.
It'll give the user more control on what to catch (if they want to catch anything).

Reply:
V